osu.Game.Rulesets.Taiko/Mods/TaikoModMirror.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Taiko.Objects;
+using osu.Game.Rulesets.UI;
+using osuTK;
+
+namespace osu.Game.Rulesets.Taiko.Mods
+{
+    public class TaikoModMirror : Mod, IApplicableToDrawableRuleset<TaikoHitObject>
+    {
+        public override string Name => "Mirror";
+        public override string Acronym => "MR";
+        public override LocalisableString Description => "Notes come from the left instead of the right.";
+        public override double ScoreMultiplier => 1;
+
+        public void ApplyToDrawableRuleset(DrawableRuleset<TaikoHitObject> drawableRuleset)
+        {
+            drawableRuleset.Scale = new Vector2(-1, 1);
```

Comment:
This is a quick and simple implementation but it also has several issues.

Judgement text mirrored:
![Screenshot 2025-05-07 at 10 01 25](https://github.com/user-attachments/assets/2b45225d-a115-4d93-98c0-ec8a58ae8e4c)

Clutter in top right on skins with mascots:
![Screenshot 2025-05-07 at 10 03 37](https://github.com/user-attachments/assets/b8df585d-4d0a-4749-a5cf-948ffe43f0cc)

I'm not sure this can be resolved amicably without some rather invasive changes, and thus my current stance is that we probably shouldn't get this mod in unless there's somehow overwhelming support for it.